### DISCUSSION TITLE
SD stops working after using Controlnet

### DISCUSSION THREAD

**Discussion by maxvector9** [Mikubill/sd-webui-controlnet#1448] [Q&A]
Since today, whenever I use controlnet to generate an image in SD, it creates an image after the first run, but after that the generate button turn orange, but doesn't respond to clicking. Also, perhaps not related, but the image generated with help of Controlnet doesn't appear in the preview box on the right. This is all using AUTOMATIC1111.

How can I fix this problem?

I'm running Controlnet v1.1.200
![Screenshot 2023-05-27 214450](https://github.com/Mikubill/sd-webui-controlnet/assets/131748915/188da266-7c6c-443c-8e8e-d692d31ed901)

**Comment by Ramoyald** [comment]
maybe its because you are using SD2.1 yaml file on a SD1.5 models?
